src/index.html

```diff
@@ -9,10 +9,72 @@
     <title>Calendar</title>
     <link
       rel="stylesheet"
-      href="styles/index.scss"
+      href="styles/main.scss"
```

Comment:
You should link to the CSS file, not the SCSS file. SCSS files are preprocessed to produce CSS, which browsers can understand. Make sure you link to the correct CSS file that is generated after preprocessing your SCSS.